tasks/get_current_archive_list/get_current_archive_list.py

```diff
@@ -429,13 +433,15 @@ def translate_s3_import_to_partitioned_data_sql() -> TextClause:
 
 def get_s3_credentials_from_secrets_manager() -> tuple:
     # todo: Move everything from here to get_configuration to shared lib. See shared_db for code origin.
-    prefix = os.getenv("PREFIX", None)
+    prefix = os.environ["PREFIX"]
     secretsmanager = boto3.client(
-        "secretsmanager", region_name=os.getenv("AWS_REGION", None)
+        "secretsmanager", region_name=os.environ["AWS_REGION"]
     )
+    secret_id = f"{prefix}-orca-{SECRETSMANAGER_S3_ACCESS_CREDENTIALS_KEY}"
```

Comment:
I would prefer to keep this code close to what we already use for DB connection info so that we can eventually use a shared library. It'll be easier to change how we interact with the SecretsManager once that is in place.

Reply:
Pass in secret id as environment variable.

Reply:
https://bugs.earthdata.nasa.gov/browse/ORCA-404